Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -646,139 +646,139 @@ Don't have a 4chan Pass?<br>
     <string name="setting_exclusion_zones_actions_dialog_title">Android 10 exclusion zones actions</string>
-    <string name="setting_exclusion_zones_wrong_phone_orientation">Wrong phone orientation, you need to rotate your phone 90 degrees to any side before using this option.</string>
+    <string name="setting_exclusion_zones_wrong_phone_orientation">Wrong device orientation. Rotate your device 90° to any side before using this option.</string>
     <string name="setting_exclusion_zones_edit_or_remove_zone_title">Edit or remove zone?</string>
     <string name="setting_exclusion_zones_zone_removed">Zone removed. Restart the app for the changes to kick in.</string>
-    <string name="setting_exclusion_zones_zone_added">New exclusion zone added. Restart the app for the changes to kick in.</string>
+    <string name="setting_exclusion_zones_zone_added">New exclusion zone added. Restart the app to use changes.</string>
 
     <string name="setting_allow_okhttp_http2">Allow OkHttp to use HTTP/2 protocol</string>
     <string name="setting_allow_okhttp_ipv6">Allow OkHttp to use IPv6</string>
-    <string name="setting_allow_okhttp_http2_ipv6_description">Disabling this setting may resolve issues with images not loading whatsoever in some rare cases. Try disabling this setting if images refuse to start loading.</string>
+    <string name="setting_allow_okhttp_http2_ipv6_description">Try turning this off if images refuse to load.</string>
 
     <string name="file_has_orientation_exif_data">This file has orientation data. Consider re-encoding it before posting.</string>
     <string name="file_type_may_not_be_supported">This file may not be supported on this imageboard. Consider re-encoding it before posting.</string>
     <string name="file_cannot_be_reencoded">Image doesn\'t support re-encoding.</string>
 
     <string name="view_my_posts">View my posts</string>
-    <string name="no_saved_posts_for_current_thread">You haven\'t posted in this thread!</string>
+    <string name="no_saved_posts_for_current_thread">None of the posts in this thread are yours.</string>
 
     <string name="linkable_copied_to_clipboard">Linkable URL copied to clipboard</string>
 
     <string name="options_button_sage">Sage</string>
     <string name="options_button_pass">Since4Pass</string>
     <string name="options_button_fortune">Fortune</string>
 
-    <string name="apply_image_filter_to_post">Apply image hash filters to posts</string>
-    <string name="apply_image_filter_to_post_description">Applies image hash filtering to posts themselves as opposed to images</string>
-    <string name="yt_api_key_dialog">Youtube API Key</string>
-    <string name="yt_api_key_dialog_description">Without a key parsing is still possible, but may not be reliable. You can use your own key, or snag the one from 4chanX.</string>
+    <string name="apply_image_filter_to_post">Image hash filtering of posts</string>
```

Comment:
Put the ``apply`` back in.

Reply:
The attempted consistency here is to have all options without accompanying verb or "enable".